assets/_scss/elements/_inputs.scss

```diff
@@ -78,6 +81,17 @@ select {
   background-position: right 1.3rem center;
 }
 
+legend {
```

Comment:
When is a legend not under a fieldset? Or is it that a legend is always supposed to be before a fieldset? Either way, it seems weird to have different rules for when a legend is in a fieldset vs when it's not.